src/test/scala/nl.knaw.dans.easy.multideposit/BlackBoxSpec.scala

```diff
@@ -19,30 +19,44 @@ import java.nio.file.{ Files, Path, Paths }
 import javax.naming.directory.{ Attributes, BasicAttribute, BasicAttributes }
 
 import org.scalamock.scalatest.MockFactory
-import org.scalatest.BeforeAndAfter
-import resource._
+import resource.managed
 
 import scala.collection.JavaConverters._
 import scala.util.{ Failure, Success }
-import scala.xml.transform.{ RewriteRule, RuleTransformer }
 import scala.xml.{ Elem, Node, NodeSeq, XML }
+import scala.xml.transform.{ RewriteRule, RuleTransformer }
 
-class BlackBoxSpec extends UnitSpec with BeforeAndAfter with MockFactory with CustomMatchers {
+// Note to developers: this classes uses shared tests as described in
+// http://www.scalatest.org/user_guide/sharing_tests
+class BlackBoxSpec extends UnitSpec with MockFactory with CustomMatchers {
```

Comment:
I've splitted this file into smaller tests, such that we can see more clearly which part of the whole test failed.